### PR TITLE
Selenium Readme

### PR DESCRIPTION
**Pull request by HonzaTuron**
Closes apify/apify-web#2676.

Replaced readme by similar to other python ones

### CODE REVIEW

**Review by nguyeda1 — APPROVED** [review]

File: templates/python-selenium/README.md [hunk 1]
```diff
@@ -1,5 +1,22 @@
-# Selenium Actor template
+# Python - Selenium & Chrome Actor template
 
-The `README.md` file documents what your actor does and how to use it, which is then displayed in the Console or Apify Store. It's always a good idea to write a `README.md`. In a few months, not even you will remember all the details about the actor.
+A template example built with Selenium and headless Chrome browser to scrape a website and save the results to storage. The URL of the web page is passed in via input, which is defined by the [input schema](https://docs.apify.com/platform/actors/development/input-schema). The template uses the [Selenium WebDriver](https://www.selenium.dev/documentation/webdriver/) to get the HTML of the page. Enqueued URLs are available in [request queue](https://docs.apify.com/sdk/python/reference/class/RequestQueue). The data are then stored in a [dataset](https://docs.apify.com/platform/storage/dataset) where you can easily access them.
+
+## Included features
+
+- **[Apify SDK](https://docs.apify.com/sdk/python/)** - a toolkit for building actors
```
Comment (jbartadev) [inline templates/python-selenium/README.md:7]:
```suggestion
- **[Apify SDK](https://docs.apify.com/sdk/python/)** - toolkit for building Apify Actors

File: templates/python-selenium/README.md [hunk 2]
```diff
@@ -1,5 +1,22 @@
...
+- **[Input schema](https://docs.apify.com/platform/actors/development/input-schema)** - define and easily validate a schema for your actor's input
+- **[Request queue](https://docs.apify.com/sdk/python/docs/concepts/storages#working-with-request-queues)** - queues into which you can put the URLs you want to scrape
+- **[Dataset](https://docs.apify.com/sdk/python/docs/concepts/storages#working-with-datasets)** - store structured data where each object stored has the same attributes
```
Comment (jbartadev) [inline templates/python-selenium/README.md:10]: How about adding the Selenium here too? It's the main feature of this template

File: templates/python-selenium/README.md [hunk 3]
```diff
@@ -1,5 +1,22 @@
...
+
+## How it works
+This code is a Python script that uses the Apify platform to scrape web pages and extract data from them. Here's a brief overview of how it works:
+
+- The script reads the input data from the Actor instance, which is expected to contain a start_urls key with a list of URLs to scrape and a max_depth key with the maximum depth of nested links to follow.
```
Comment (jbartadev) [inline templates/python-selenium/README.md:15]: Let's use the `code` formatting for the variable names everywhere in the description (`start_urls`, `max_depth`,..) - it makes reading easier and users would know you're referencing to something in the code

**Review by jbartadev — CHANGES_REQUESTED** [review]
Small details

File: templates/python-selenium/README.md [hunk 4]
```diff
@@ -1,5 +1,22 @@
-# Selenium Actor template
+# Python - Selenium & Chrome Actor template
 
-The `README.md` file documents what your actor does and how to use it, which is then displayed in the Console or Apify Store. It's always a good idea to write a `README.md`. In a few months, not even you will remember all the details about the actor.
+A template example built with Selenium and headless Chrome browser to scrape a website and save the results to storage. The URL of the web page is passed in via input, which is defined by the [input schema](https://docs.apify.com/platform/actors/development/input-schema). The template uses the [Selenium WebDriver](https://www.selenium.dev/documentation/webdriver/) to get the HTML of the page. Enqueued URLs are available in [request queue](https://docs.apify.com/sdk/python/reference/class/RequestQueue). The data are then stored in a [dataset](https://docs.apify.com/platform/storage/dataset) where you can easily access them.
```
Comment (fnesveda) [inline templates/python-selenium/README.md:3]:
```suggestion
A template example built with Selenium and headless Chrome browser to scrape a website and save the results to storage. The URL of the web page is passed in via input, which is defined by the [input schema](https://docs.apify.com/platform/actors/development/input-schema). The template uses the [Selenium WebDriver](https://www.selenium.dev/documentation/webdriver/) to load and process the page. Enqueued URLs are stored in the default [request queue](https://docs.apify.com/sdk/python/reference/class/RequestQueue). The data are then stored in the default [dataset](https://docs.apify.com/platform/storage/dataset) where you can easily access them.
```

File: templates/python-selenium/README.md [hunk 5]
```diff
@@ -1,5 +1,22 @@
...
+
+## Included features
+
+- **[Apify SDK](https://docs.apify.com/sdk/python/)** - a toolkit for building actors
+- **[Input schema](https://docs.apify.com/platform/actors/development/input-schema)** - define and easily validate a schema for your actor's input
```
Comment (fnesveda) [inline templates/python-selenium/README.md:8]:
```suggestion
- **[Input schema](https://docs.apify.com/platform/actors/development/input-schema)** - define and easily validate a schema for your Actor's input
```

File: templates/python-selenium/README.md [hunk 6]
```diff
@@ -1,5 +1,22 @@
...
+- **[Dataset](https://docs.apify.com/sdk/python/docs/concepts/storages#working-with-datasets)** - store structured data where each object stored has the same attributes
+
+## How it works
+This code is a Python script that uses the Apify platform to scrape web pages and extract data from them. Here's a brief overview of how it works:
+
+- The script reads the input data from the Actor instance, which is expected to contain a start_urls key with a list of URLs to scrape and a max_depth key with the maximum depth of nested links to follow.
+- The script enqueues the starting URLs in the default request queue and sets their depth to 1.
+- The script processes the requests in the queue one by one, fetching the URL using requests and parsing it using Selenium.
+- If the depth of the current request is less than the maximum depth, the script looks for nested links in the page and enqueues their targets in the request queue with an incremented depth.
+- The script extracts the desired data from the page (in this case, titles of each page) and pushes it to the default dataset using the push_data method of the Actor instance.
```
Comment (fnesveda) [inline templates/python-selenium/README.md:19]:
```suggestion
- The script extracts the desired data from the page (in this case, titles of each page) and pushes them to the default dataset using the push_data method of the Actor instance.
```

File: templates/python-selenium/README.md [hunk 7]
```diff
@@ -1,5 +1,22 @@
...
+- The script catches any exceptions that occur during the scraping process and logs an error message using the Actor.log.exception method.
+- This code demonstrates how to use Python and the Apify SDK to scrape web pages and extract specific data from them.
```
Comment (fnesveda) [inline templates/python-selenium/README.md:21]: This is kind of redundant, same as line 13

**Review by fnesveda — CHANGES_REQUESTED** [review]

File: templates/python-selenium/README.md (hunk 2 above)
Comment (HonzaTuron) [reply templates/python-selenium/README.md:10]: I thought this was generally related to Apify features, at least that's how it works in other templates. Also, it's mentioned in paragraph above.

File: templates/python-selenium/README.md [hunk 8]
```diff
@@ -1,5 +1,20 @@
...
-You can use [Markdown](https://www.markdownguide.org/cheat-sheet) language for rich formatting.
+## Included features
+
+- **[Apify SDK](https://docs.apify.com/sdk/python/)** - toolkit for building Apify Actors
+- **[Input schema](https://docs.apify.com/platform/actors/development/input-schema)** - define and easily validate a schema for your Actor's input
+- **[Request queue](https://docs.apify.com/sdk/python/docs/concepts/storages#working-with-request-queues)** - queues into which you can put the URLs you want to scrape
+- **[Dataset](https://docs.apify.com/sdk/python/docs/concepts/storages#working-with-datasets)** - store structured data where each object stored has the same attributes
+
+## How it works
+This code is a Python script that uses the Apify platform to scrape web pages and extract data from them. Here's a brief overview of how it works:
```
Comment (fnesveda) [inline templates/python-selenium/README.md:13]: This is not really true, the script doesn't use the Apify platform if you run it locally. I would replace "Apify platform" with "Selenium" here.

**Review by fnesveda — CHANGES_REQUESTED** [review]
One more note

File: templates/python-selenium/README.md [hunk 9]
```diff
@@ -1,5 +1,20 @@
-# Selenium Actor template
+# Python - Selenium & Chrome Actor template
```
Comment (jbartadev) [inline templates/python-selenium/README.md:1]: I don't think it needs to have the `Python` prefix here, we don't have that pattern for other templates. In CLI you need to choose the language so you already know it's Python templates and on template detail we have it the category in the sidebar.

```suggestion
# Selenium & Chrome Actor template
```

File: templates/python-selenium/README.md [hunk 10]
```diff
@@ -1,5 +1,20 @@
...
-You can use [Markdown](https://www.markdownguide.org/cheat-sheet) language for rich formatting.
+## Included features
+
+- **[Apify SDK](https://docs.apify.com/sdk/python/)** - toolkit for building Apify Actors
+- **[Input schema](https://docs.apify.com/platform/actors/development/input-schema)** - define and easily validate a schema for your Actor's input
+- **[Request queue](https://docs.apify.com/sdk/python/docs/concepts/storages#working-with-request-queues)** - queues into which you can put the URLs you want to scrape
+- **[Dataset](https://docs.apify.com/sdk/python/docs/concepts/storages#working-with-datasets)** - store structured data where each object stored has the same attributes
+
+## How it works
+This code is a Python script that uses the Selenium to scrape web pages and extract data from them. Here's a brief overview of how it works:
```
Comment (fnesveda) [inline templates/python-selenium/README.md:13]:
```suggestion
This code is a Python script that uses Selenium to scrape web pages and extract data from them. Here's a brief overview of how it works:
```

**Review by fnesveda — CHANGES_REQUESTED** [review]
One more article from the latest change

**Review by jbartadev — APPROVED** [review]

**Pull request merged**